playwright/e2e/create-colony.spec.ts

```diff
@@ -0,0 +1,388 @@
+      await expect(
+        page.getByRole('button', { name: /continue/i }),
+      ).toBeEnabled();
+    });
+  });
+
+  test.describe('Confirmation step', () => {
+    test.beforeEach(async ({ page }) => {
+      await fillColonyNameStep(page, {
+        nameFieldValue: colonyName,
+        urlFieldValue: customColonyURL,
+      });
+    });
+
+    test('Should render Confirmation step as expected', async ({ page }) => {
+      await fillNativeTokenStepWithExistingToken(page, validToken);
+      // Heading of the Step
+      await expect(
+        page.getByText(/Confirm your Colony’s details/i),
+      ).toBeVisible();
+      // Subheading
+      await expect(
+        page.getByText(
+          /Check to ensure your Colony’s details are correct as they can not be changed later/i,
+        ),
+      ).toBeVisible();
+
+      const colonyNameCard = page.getByTestId('colony-details-card');
+      const colonyTokenCard = page.getByTestId('colony-token-card');
+
+      await expect(colonyNameCard.getByText('testcolonyname')).toBeVisible();
+
+      await expect(
+        colonyNameCard.getByRole('button', { name: 'Edit' }),
+      ).toBeEnabled();
+
+      await expect(
+        colonyTokenCard.getByText('DAI for Local Development'),
+      ).toBeVisible();
+
+      await expect(
+        colonyTokenCard.getByRole('button', { name: 'Edit' }),
+      ).toBeEnabled();
+
+      await expect(page.getByRole('button', { name: 'Back' })).toBeVisible();
+
+      await expect(
+        page.getByRole('img', {
+          name: /Avatar of token DAI for Local Development/i,
+        }),
+      ).toBeVisible();
+
+      await expect(
+        page.getByRole('button', { name: /continue/i }),
+      ).toBeEnabled();
+    });
+
+    test('Should navigate to Details step when Edit colony details is clicked', async ({
+      page,
+    }) => {
+      await fillNativeTokenStepWithExistingToken(page, validToken);
+
+      await page
+        .getByTestId('colony-details-card')
+        .getByRole('button', { name: 'Edit' })
+        .click();
+
+      await expect(page.getByLabel(/Colony name/i)).toBeVisible();
+      await expect(page.getByLabel(/Colony name/i)).toHaveValue(colonyName);
+      await expect(page.getByLabel(/Custom Colony URL/i)).toBeVisible();
+      await expect(page.getByLabel(/Custom Colony URL/i)).toHaveValue(
+        customColonyURL,
+      );
+    });
+
+    test('Should navigate to Native token step when Edit token is clicked', async ({
+      page,
+    }) => {
+      await fillNativeTokenStepWithExistingToken(page, validToken);
+
+      await page
+        .getByTestId('colony-token-card')
+        .getByRole('button', { name: 'Edit' })
+        .click();
+
+      await expect(page.getByLabel(/Existing token address/i)).toBeVisible();
+      await expect(page.getByLabel(/Existing token address/i)).toHaveValue(
+        validToken,
+      );
+    });
+
+    test('Should create a colony and navigate to the Complete Setup step', async ({
```

Comment:
At this stage the colony is not yet created, so this description is a bit misleading.

I would actually have this step wait until all transactions are complete and you get the "successfully created colony" welcome modal _(although that might be harder)_

Reply:
added this step, although it's quite slow as the transactions and  transition to the new colony page take some time